css-uc/styles.css

```diff
@@ -0,0 +1,324 @@
+/* General properties */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --white: #FFFFFF;
+  --blue-gradient: linear-gradient(46.99deg, #0F58B7 -4.4%, #62D9FF 120.42%);
+  --black: #1C1C1C;
+  --lightblue-gradient: linear-gradient(46.77deg, #0F99B7 -0.79%, #69ECDC 131.52%);
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Poppins';
+  background-color: var(--white);
+  margin-inline: auto;
+  width: 80%;
+}
+
+/* Flex properties*/
+.flex {
+  display: flex;
+}
+
+.flex-col {
+  flex-direction: column;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.justify-around {
+  justify-content: space-around;
+}
+
+.justify-between {
+  justify-content: space-between;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.items-center {
+  align-items: center;
+}
+
+.items-start {
+  align-items: flex-start;
+}
+
+/* Margins, paddings and gaps */
+.mt-1 {
+  margin-top: 10px;
+}
+
+.mt-7 {
+  margin-top: 70px;
+}
+
+.mt-6 {
+  margin-top: 60px;
+}
+
+.mt-14 {
+  margin-top: 140px;
+}
+
+.mt-22 {
+  margin-top: 220px;
+}
+
+.p-btn1 {
+  padding: 5px 40px;
+}
+
+.p-btn2 {
+  padding: 9.5px 53px;
+}
+
+.p-btn3 {
+  padding: 8px 40px;
+}
+
+.gap-1 {
+  gap: 10px;
+}
+
+.gap-2 {
+  gap: 20px;
+}
+
+.gap-3 {
+  gap: 30px;
+}
+
+.gap-4 {
+  gap: 40px;
+}
+
+.gap-6 {
+  gap: 60px;
+}
+
+.gap-15 {
+  gap: 150px;
+}
+
+.gap-20 {
+  gap: 200px
+}
+
+/* Lists properties */
+.list-none {
+  list-style: none;
+}
+
+/* Text sizes, colors, align and letter spacing*/
+.text-2xl {
+  font-size: 6.0rem;
+}
+
+.text-xl {
+  font-size: 4.8rem;
+}
+
+.text-lg {
+  font-size: 3.5rem;
+}
+
+.text-sm {
+  font-size: 3.0rem;
+}
+
+.text-base {
+  font-size: 1.6rem;
+}
+
+.tracking-tight {
+  letter-spacing: -1px;
+}
+
+.leading-tight {
+  line-height: 108%;
+}
+
+.leading-snug {
+  line-height: 130%;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.font-semibold {
+  font-weight: 600;
+}
+
+.font-medium {
+  font-weight: 500;
+}
+
+.font-light {
+  font-weight: 300;
+}
+
+.text-white {
+  color: var(--white);
+}
+
+.text-blue-descendent {
+  background: var(--blue-gradient);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+
+.text-black {
+  color: var(--black);
+}
+
+.text-fill-black {
+  -webkit-text-fill-color: var(--black);
+}
+
+.no-underline {
+  text-decoration: none;
+}
+
+/* Widths */
+.w-90 {
+  width: 90%;
+}
+
+.w-80 {
+  width: 80%;
+}
+
+.w-60 {
+  width: 60%;
+}
+
+.w-55 {
+  width: 55%;
+}
+
+.w-50 {
+  width: 50%;
+}
+
+/* Container and images sizes */
+.size4 {
+  width: 94px;
+  height: 25px;
+}
+
+.img-size-xl {
+  width: 466px;
+  height: 323px;
+}
+
+.img-size-l {
+  width: 222.62px;
+  height: 270.73px;
+}
+
+.img-size-r {
+  width: 222.62px;
+  height: 192px;
+}
+
+.img-size-s {
+  width: 130.69px;
+  height: 43.31px;
+}
+
+.img-size-xs {
+  width: 222.62px;
+  height: 231.39px;
+}
+
+.img-size-xxs {
+  width: 30px;
+  height: 30px;
+}
```

Comment:
Let's start making use of responsive images here instead of uisng fixed width and height